test/metabase/models/card_test.clj

```diff
@@ -327,6 +328,35 @@
        (is (= expected
               (t2/select-one-fn :visualization_settings :model/Card :id (u/the-id card))))))))
 
+(deftest template-tag-parameters-test
+  (testing "Card with a Field filter parameter"
+    (t2.with-temp/with-temp [:model/Card card {:dataset_query (qp.card-test/field-filter-query)}]
```

Comment:
```suggestion
    (mt/with-temp [:model/Card card {:dataset_query (qp.card-test/field-filter-query)}]
```'

we should use `mt/with-temp` moving forward. It makes future refactoring easier in case we want to change how with-temp works internally.

Reply:
Done a1e7cb5ad37a4a22a2fc2027de2498ee72cbe4f5